python/mxnet/metric.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, name, output_names=None,
         self.output_names = output_names
         self.label_names = label_names
         self._kwargs = kwargs
+        if "has_global_stats" in kwargs:
+            self._has_global_stats = kwargs["has_global_stats"]
+        else:
+            self._has_global_stats = False
```

Comment:
`self._has_global_stats = kwargs.get("has_global_stats", False)` ?

Reply:
done (with pop to not keep the has_global_stats key in kwargs, to not screw with deserialization later.)